src/settings-ui/Settings.UI/SettingsXAML/OOBE/Views/OobeRegistryPreview.xaml.cs

```diff
@@ -27,10 +27,10 @@ public OobeRegistryPreview()
 
         private void Launch_RegistryPreview_Click(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
         {
-            ShellPage.SendDefaultIPCMessage("{\"action\":{\"RegistryPreview\":{\"action_name\":\"Launch\", \"value\":\"\"}}}");
+            ShellPage.SendDefaultIPCMessage("{\"action\":{\"RegistryPreview\":{\"action_name\":\"Open\", \"value\":\"\"}}}");
```

Comment:
@jaimecbernardo I wonder if this will be a "breaking change" too?

Reply:
You're right, it might be 🤔 Let's leave it as launch, I guess?

Reply:
Already done after your comment, check out the latest commits :)